app/packages/core/src/components/Sidebar/Entries/PathValueEntry.tsx

```diff
@@ -244,17 +244,25 @@ const useData = <T extends unknown>(path: string): T => {
 
   let field = useRecoilValue(fos.field(keys[0]));
 
-  for (let index = 0; index < keys.length; index++) {
-    if (!data) {
-      break;
-    }
+  // assume allowing the user to use one layer dynamic embedded doc fields
+  if (
+    field?.embeddedDocType ==
+    "fiftyone.core.odm.embedded_document.DynamicEmbeddedDocument"
```

Comment:
Yeah, I recommend adding this to `@fiftyone/utilties` as `DYNAMIC_EMBEDDED_DOCUMENT` or similary